gatsby-node.js

```diff
@@ -124,6 +127,14 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
 
   const members = res.data.allMembers.nodes;
 
+  paginate({
+    createPage,
+    items: blogs,
+    itemsPerPage: 8,
+    pathPrefix: `/blog`,
+    component: blogViewTemplate

```

Comment:
Will we need to update this code in the future to accommodate news, workshops, and other collections? If so, is there a way to make this less fragile?

Reply:
The paginate function would need to be called for each of the collections in that case

Reply:
Ok, a centrally-controlled configuration to enable pagination on specific collections. This makes sense.